### ISSUE TITLE
install script isn't working

### ISSUE THREAD

**Issue by brian-chilin** [k3s-io/k3s#8736]
I have two pis and I'd like to try making a cluster. I'm struggling to even get a master node working. 

Raspberry Pi 3 Model B+ with Raspbian 11 32bit Lite
Raspberry Pi 2 Model B V1.1 with Raspbian Raspbian 11 32bit Lite

`uname -m`
`armv7l`

I'm trying to install k3 with
`curl -sfL https://get.k3s.io | sh - `

And it outputs
```
brian@raspi31:~ $ curl -sfL https://get.k3s.io | sh -
[INFO]  Finding release for channel stable
curl: (7) Couldn't connect to server
[INFO]  Using v1.27.6+k3s1 as release
[INFO]  Downloading hash https://github.com/k3s-io/k3s/releases/download/v1.27.6+k3s1/sha256sum-arm.txt
```

Internet seems to be working and I've added `cgroup_memory=1 cgroup_enable=memory` to /boot/cmdline.txt
I'm unsure if I'm missing something or doing something wrong.

**Comment by brian-chilin** [comment]
this is a problem with other stuff like my dns resolution or something

for some sites
like ping google.com or ping youtube.com works

ping github.com or ping amazon.com don't work

**Comment by brian-chilin** [comment]
this wound up being my something about my default gateway or router

use `route -n` to see gateways

`sudo nano /etc/dhcpcd.conf`
edit, uncomment, add, or fix a line, I needed `static routers=192.168.1.1`
I think it was by deafult 192.168.0.1

`sudo systemct restart dhcpcd.service`, i opted to restart instead

not really sure why this is, what was wrong, or why this fixed it
